hatServer/static/js/services/surveyQuestions.js

```diff
@@ -0,0 +1,461 @@
+'use strict';
+
+sortingApp.service('surveyQuestions', function () {
+    var surveyName = "";
+    var surveyDescription = 'HELLO JESSICA YOU DA BEST';
+    var desiredSkills = ['Java', 'Python', 'PHP', 'C/C++', 'Mobile App Development (Android/iOS)', 'Web Applications', 'Embedded Systems', 'Database (MySQL, SQL, etc.)', 'User Interface/Experience', 'Statistics', 'Networking',
+        'Robotics', 'Computer Vision', 'Algorithms', 'Machine Learning'];
```

Comment:
I don't know if there's a convenient way to do this, but we need to make sure these skills are consistent with the list the instructors use when building the groups.